libs/payments/stripe/src/lib/stripe.client.types.ts

```diff
@@ -252,7 +252,7 @@ export type StripePlan = NegotiateExpanded<
  */
 export type StripePrice = NegotiateExpanded<
   never,
-  Stripe.Price,
+  DeepOverride<Stripe.Price, { recurring: Stripe.Price.Recurring }>,
```

Comment:
This DeepOverride isn't correct - recurring is not always present on a price according to the docs (one-time payments)